cluster/src/test/scala/org/apache/spark/sql/store/BugTest.scala

```diff
@@ -475,5 +475,68 @@ class BugTest extends SnappyFunSuite with BeforeAndAfterAll {
 
   }
 
+  test("Bug SNAP-2887") {
```

Comment:
The intention of the test just looks sanity on inserts. Do you wish to fire queries which uses push down when conditions are on indexed or pk columns?

Reply:
removed those.